src/atsu/system/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import { Atsu } from './ATSU';
```

Comment:
We can replace those with `export * from './X'`

Reply:
fixed